src/dashboard/src/main/templatetags/lang.py

```diff
@@ -0,0 +1,13 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def standardize_lang_code(language_code):
+    """Django uses language codes like es-es or pt-br. This will convert them
+    into es_ES and pt_BR."""
+    head, sep, tail = language_code.partition("-")
+    if sep == '':
+        return head
+    return "{}_{}".format(head, tail.upper())
```

Comment:
I had to do this so `pt-br` shows up as `pt_BR` in `window.DashboadConfig` - otherwise `angular-gettext` wouldn't understand the value as it expects the latter format.